msys2-runtime/PKGBUILD

```diff
@@ -65,7 +65,8 @@ source=('msys2-runtime'::git://sourceware.org/git/newlib-cygwin.git#tag=cygwin-$
         0040-Emulate-GenerateConsoleCtrlEvent-upon-Ctrl-C.patch
         0041-kill-kill-Win32-processes-more-gently.patch
         0042-Fix-64-vs-32-bit-type-confusion-in-swprintf.patch
-        0043-Don-t-fall-back-to-ExitProcess-if-process-handled-Ct.patch)
+        0043-Don-t-fall-back-to-ExitProcess-if-process-handled-Ct.patch
+				0044-getprocaddr-refactor-cleanup.patch)
```

Comment:
Oops, looks like my vim config got screwed up and put tabs instead of spaces.  Hopefully will get cleaned up in next update.